### ISSUE TITLE
Feature request: 'Open with… [Tagger]' option

### ISSUE THREAD

**Issue by rene-coty** [NickvisionApps/Tagger#267]
I would like to be able to open one or several files I want to edit with Tagger _directly_ from Nautilus / my file manager. Currently you have to open Tagger first and then select a directory which is not very intuitive (e.g. Paper Clip - a PDF metadata editor - lets you open files from the file manager, same for Metadata Cleaner…)

Could be related to #259 as opening one /several files in the app without going through the directory chooser dialog seems to go against the current logic of Tagger

**Comment by nlogozzo** [comment]
Do you use Tagger via flatpak or installed manually/aur? Beside this feature will not work on Flatpak due to its sandboxing nature...but if you use AUR or something like that I can implement this feature for that...

**Comment by fsobolev** [comment]
> Beside this feature will not work on Flatpak due to its sandboxing nature...but if you use AUR or something like that I can implement this feature for that...

I think it worth clarifying that the problem is not the ability to provide the feature in Flatpak, we could provide mimetype handler for file managers to consume just like we do in Denaro, but the problem is Tagger currently can only access xdg-music and xdg-dowload directories without portal.
So the question arise again: should we expect users to open files outside of xdg-music and xdg-download in Tagger? If not, then I think opening files from file manager brings no harm, in worst - unexpected - case we could show a toast about insufficient permissions. But if it's not enough, I guess we have to add home permission.

**Comment by fsobolev** [comment]
Implemented now, and we added home permission, but take note that if you will try to open a folder outside of home using "Open with..." then Tagger will not start. We can't do anything about this, it's [flatpak bug](https://www.github.com/flatpak/flatpak/issues/4799). If you have music files outside of home, it's recommended to give Tagger permission for required folder using Flatseal (not only for "Open with..." to work, but also to get better performance).
